From ad9f1487cfb3ce4e90fa7f6f2344921b806f9ecc Mon Sep 17 00:00:00 2001
From: Tomas Glozar <tglozar@gmail.com>
Date: Fri, 20 Mar 2026 20:32:21 +0100
Subject: [PATCH 7/8] ia64: Add listns() syscall

listns() was added after ia64 was removed from the kernel.

Fixes: b36d4b6aa88e ("arch: hookup listns() system call")
Signed-off-by: Tomas Glozar <tglozar@gmail.com>
---
 arch/ia64/kernel/syscalls/syscall.tbl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/arch/ia64/kernel/syscalls/syscall.tbl b/arch/ia64/kernel/syscalls/syscall.tbl
index 75b3d45c4815..36c2d6cba49b 100644
--- a/arch/ia64/kernel/syscalls/syscall.tbl
+++ b/arch/ia64/kernel/syscalls/syscall.tbl
@@ -390,3 +390,4 @@
 467	common	open_tree_attr			sys_open_tree_attr
 468	common	file_getattr			sys_file_getattr
 469	common	file_setattr			sys_file_setattr
+470	common	listns				sys_listns
-- 
2.52.0

